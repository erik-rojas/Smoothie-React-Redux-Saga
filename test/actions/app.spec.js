import { hideAlert, showAlert } from 'actions/app';

describe('App', () => {
  it('showAlert with type `error` should return an action', () => {
    expect(showAlert('Alright!', { id: 'test', type: 'error' })).toMatchSnapshot();
  });

  it('showAlert with type `success` should return an action', () => {
    expect(showAlert('Alright!', { id: 'test', type: 'success', timeout: 10 })).toMatchSnapshot();
  });

  it('hideAlert should return an action', () => {
    expect(hideAlert('test')).toMatchSnapshot();
  });
});
