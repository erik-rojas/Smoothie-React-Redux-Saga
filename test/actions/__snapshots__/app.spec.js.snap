// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App hideAlert should return an action 1`] = `
Object {
  "payload": Object {
    "id": "test",
  },
  "type": "HIDE_ALERT",
}
`;

exports[`App showAlert with type \`error\` should return an action 1`] = `
Object {
  "payload": Object {
    "icon": undefined,
    "id": "test",
    "message": "Alright!",
    "position": "bottom-right",
    "timeout": 0,
    "type": "error",
  },
  "type": "SHOW_ALERT",
}
`;

exports[`App showAlert with type \`success\` should return an action 1`] = `
Object {
  "payload": Object {
    "icon": undefined,
    "id": "test",
    "message": "Alright!",
    "position": "bottom-right",
    "timeout": 10,
    "type": "success",
  },
  "type": "SHOW_ALERT",
}
`;
